### PR TITLE
docs: add contribute guide

### PR DESCRIPTION
**Pull request by kecrily**
It fixes a bug in `eslintVersion.js` that would make the site build fail. So don't worry about the build failures in #2 #3 #4, they will all go back to normal after this PR merge.

### CODE REVIEW

**Comment by kecrily** [comment]
@nzakas I can't merge the PR because it requires only one approval. Should I continue to wait for the Chinese reviewer to review it, or commit it directly to the `main` branch, or something else?

File: package.json
Comment (nzakas) [inline package.json:2]: This needs to be zh-hans.docs.eslint.org because the docs need to be deployed to their own domain separate from the main site.

**Review by nzakas — CHANGES_REQUESTED** [review]

File: package.json
Comment (kecrily) [reply package.json:2]: I thought it would be deployed in the subdirectory same as the main site. Ok, I will revert it.

**Review by nzakas — APPROVED** [review]

**Pull request merged**
